packages/design/jobberStyle.js

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const fs = require("fs");
+// eslint-disable-next-line import/no-internal-modules
+const customPropertiesObject = require("./foundation.js");
+
+const customProperties = customPropertiesObject.customProperties;
+const allKeys = Object.keys(customProperties);
+const resolvedCssVars = allKeys.reduce((acc, key) => {
+  const newKey = key.replace("--", "");
+  acc[newKey] = jobberStyle(key);
+  return acc;
+}, {});
+
+const jsonContent =
+  "export const JobberStyle = " + JSON.stringify(resolvedCssVars, undefined, 2);
+
+fs.writeFile("./foundation.js", jsonContent, "utf8", function(err) {
+  if (err) {
+    console.log("An error occured while writing JSON object to File.");
+    return console.log(err);
+  }
+  console.log("JSON file has been saved.");
+});
+
+function jobberStyle(styling) {
+  const varRegExp = /var\((.*)\)/;
+  const calcRegExp = /calc\((.*)\)/;
+  const removeAllNonNumerals = /[^0-9.+\-/*]/gi;
+  const styleValue = customProperties[styling];
+  const varRegexResult = varRegExp.exec(styleValue);
```

Comment:
This is a really unhelpful comment but, I'm finding this method a bit hard to parse. I wonder if it would be more readable if broken into smaller more functional bits?